recipes/implot/all/conanfile.py

```diff
@@ -1,20 +1,23 @@
-from conans import ConanFile, CMake, tools
-import functools
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import get, copy
+from conan.tools.scm import Version
+import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.59"
```

Comment:
We could get away with less

```suggestion
required_conan_version = ">=1.54.0"
```

Reply:
Done in [1f87f2c](https://github.com/conan-io/conan-center-index/pull/16847/commits/1f87f2c5f21961b28d7bc610813495df3170c5e0)